frontend/src/components/footer.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
```

Comment:
This was moved to `components/common/footer/index.jsx`; please pull any relevant code from here into that file and delete this file.